src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
+    white,
+    red,
+    blue
```

Comment:
these are constants, and according to code convention constants must be named caps lock
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names